README.md

```diff
@@ -48,26 +73,47 @@ val rdd = sc.parallelize(Array(Array("foo", "bar"), Array("baz", "qux")))
 rdd.saveToMemsql(dbHost, dbPort, dbUser, dbPassword, dbName, outputTableName, insertBatchSize=1000)
 ```
 
-MemSQL Spark ETL
-----------------
-The MemSQL Spark ETL library provides tools for defining a Spark Streaming job that
-integrates with MemSQL Ops.
+Using
+-----
 
-Examples
---------
+The recommended way to depend on the above projects is via our private Maven repository [maven.memsql.com](maven.memsql.com).  You can do this in your build.sbt file for [Simple Build
+Tool (aka sbt)](http://www.scala-sbt.org/) integration.  Add the following line at the top level of your build.sbt to add our repo:
 
-Building
---------
-Run ``make package`` to compile this connector.  This will create a
-directory called ``distribution/dist/memsql-<version number>``, which will
-contain a .jar file.  Simply put this .jar file in your class path to
-use this library.
+```
+resolvers += "memsql" at "http://maven.memsql.com"
+```
 
-This directory will also contain HTML documentation for the library.
+Then inside a project definition you can depend on our libraries like so:
+
+```
+libraryDependencies  += "com.memsql" %% "memsqletl" % "VERSION"
+```
+
+We also support packaging the MemSQL Spark Connector from source. Run ``make package`` to compile this connector.  This will create a directory called ``distribution/dist/memsql-<version number>``, which will contain a .jar file. Simply put this .jar file in your class path to use this library.
 
 In order to compile this library you must have the [Simple Build
 Tool (aka sbt)](http://www.scala-sbt.org/) installed.
 
+Building
+--------
+
+You can use SBT to compile all of the projects in this repo.  To build all of the projects you can use:
+
+```
+sbt "project etlLib" build \
+    "project connectorLib" build \
+    "project interface" build
+```
+
+Testing
+-------
+
+All unit tests can be run via sbt.  They will also run at build time automatically.
+
+```
+sbt test
```

Comment:
You actually need to specify the project here, I think, e.g. `sbt "project interface" test`.